addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py

```diff
@@ -753,3 +761,6 @@ def _import_retrieve_info_from_map(self, tree, import_method_map):
                 return record
 
         return None
+
+    def _get_currency_decimal_places(self, record):
```

Comment:
I would move it far higher in the file, as you use it for the export, with the other getter for export (https://github.com/odoo/odoo/blob/1b27a19f10357c240b114508df506cdb38b7c72a/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py#L15) instead of for import (https://github.com/odoo/odoo/blob/1b27a19f10357c240b114508df506cdb38b7c72a/addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py#L688)

Reply:
You are right, I will go even further by putting in in edi.common